{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -90,10 +90,10 @@ class ProjectDefault(Configuration):
 
     # Password validation
     # https://docs.djangoproject.com/en/stable/ref/settings/#auth-password-validators
-    # fmt: off
+
     AUTH_PASSWORD_VALIDATORS = [
         {
-            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",  # noqa
+            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator", # noqa: E501
```

Comment:
```suggestion
            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",  # noqa: E501
```
Leave two spaces before comment

Reply:
okk